src/api/assessment/content-types/survey/schema.json

```diff
@@ -0,0 +1,36 @@
+
+{
+  "kind": "collectionType",
+  "collectionName": "surveys",
+  "info": {
+    "singularName": "survey",
+    "pluralName": "surveys",
+    "displayName": "Assessment.Survey"
+  },
+  "options": {
+    "draftAndPublish": true,
+    "comment": ""
+  },
+  "attributes": {
+    "slug": {
```

Comment:
* **note:** strapi automatically sets up a default `id` field so we shouldn't need to port these over from the Gatsby types
* **note:** `slug` is usually a url-friendly version of a name or title, which websites use to map human-readable urls to the underlying data they correspond to (i.e. my-region-name => region # 12 in the database).  it doesn't make much sense as the label for a UID field
* **suggestion(blocking):** remove this attribute

Reply:
The attribute has been removed.